contrib/opbot/botmd/commands.go

```diff
@@ -377,32 +392,41 @@ func (b *Bot) rfqRefund() *slacker.CommandDefinition {
 	}
 }
 
-func (b *Bot) makeFastBridge(ctx context.Context, req *relapi.GetQuoteRequestResponse) (*fastbridge.FastBridge, error) {
+// returns origin and dest fast bridge contracts
+func (b *Bot) makeFastBridge(ctx context.Context, req *relapi.GetQuoteRequestResponse) (*fastbridge.FastBridge, *fastbridge.FastBridge, error) {
 	client, err := rfqClient.NewUnauthenticatedClient(b.handler, b.cfg.RFQApiURL)
 	if err != nil {
-		return nil, fmt.Errorf("error creating rfq client: %w", err)
+		return nil, nil, fmt.Errorf("error creating rfq client: %w", err)
 	}
 
 	contracts, err := client.GetRFQContracts(ctx)
 	if err != nil {
-		return nil, fmt.Errorf("error fetching rfq contracts: %w", err)
+		return nil, nil, fmt.Errorf("error fetching rfq contracts: %w", err)
 	}
 
-	chainClient, err := b.rpcClient.GetChainClient(ctx, int(req.OriginChainID))
-	if err != nil {
-		return nil, fmt.Errorf("error getting chain client: %w", err)
-	}
+	var fastBridges [2]*fastbridge.FastBridge
+	chainIDs := []uint32{req.OriginChainID, req.DestChainID}
```

Comment:
Now that I reread my initial comment about making two contract instances, this does feel a bit clunky. Do you think that passing chain ID instead of `GetQuoteRequestResponse` would be cleaner/more generic (and making a single FastBridge instance for it)?